engine/sim/profileset.cpp

```diff
@@ -137,6 +138,10 @@ void simulate_profileset( sim_t* parent, profileset::profile_set_t& set, sim_t*&
 
   if ( !ret || profile_sim -> is_canceled() )
   {
+    if ( profile_sim->culled )
+    {
+      fmt::print( stderr, "\nProfileset '{}' culled: {}\n", set.name(), profile_sim->culled_reason );
+    }
```

Comment:
Is there a reason why you kept this one around? If the profileset sim ends in an error or is forcefully cancelled, I don't think we can say it was culled. You can refer to my earlier comment on this.

Reply:
Nope - this was cruft. Thank you.